README.md

```diff
@@ -54,9 +54,34 @@ Copy this JSON. Now, switch to "Send Event" action view. You'll likely won't see
+1. Running `npm version major/minor/patch` should have changed `package.json` and `package-lock.json`, committed the changes, and created a new git tag. Push these changes to Github by running `git push origin master --follow-tags`. Go to Github and ensure you see the newly added commit and tag.
+1. Create a new production build of Alloy by running `npm run build:prod`.
+1. Open up `dist/reactor/alloy.js` and copy the file's contents.
+1. Open the Alloy extension repository (the one you're looking at right now) locally and make sure you're on the `master` branch and have pulled the latest changes.
+1. Open `src/lib/runAlloy.js`.
+1. Find the `// LIBRARY CODE` comment and the `// END OF LIBRARY CODE` comment. Replace the content between the two comments with the content you previously copied from the Alloy repository.
+1. Manually bump the version in `extension.json`. Follow the same semantic versioning rules that you did for Alloy. The version used in the extension does not need to match the version used in Alloy. In fact, they'll almost always be different because sometimes we'll make changes in the Alloy Launch extension that don't require changes in Alloy.
+1. At this point, it's good to test out the latest code before committing your local changes. Start by running `npm run package` to create a zip file you'll upload to Launch.
+1. Run `npx @adobe/reactor-uploader` to upload the zip file to Launch. You'll likely find it easiest to pass arguments and use environment variables as outlined in the [reactor-uploader documentation](https://www.npmjs.com/package/@adobe/reactor-uploader). This information will come from the Adobe I/O integration you previously created. You may even want to create a bash script that you can easily re-run in the future, but don't commit it to the git repository because the argument values will be specific to your Adobe I/O integration.
+1. Once the zip file has been uploaded and processed, the new version will be available only to properties configured for extension development within the Unified JS Lab organization. Go create a property configured for extension development within the Unified JS Lab within Launch, install the extension, create some rules, create a library containing all the resources, and build the library. Then, place the Launch dev environment's embed code on a test page and test that your rules run properly. These steps are documented in more detail in the [Adobe Launch developer documentation](https://developer.adobelaunch.com/extensions/submissions/upload-and-test/#4-create-a-development-property).
```

Comment:
We might want to all use the same Launch property and have extension_index.html (that's in Alloy's sandbox directory) use the library from that Launch property.  I personally have been manually testing using a page on my own domain (http://dtm.aaronhardy.com/launch/alloy.html).